### ISSUE TITLE
attribute in let block - used therein also - is deleted

### ISSUE THREAD

**Issue by 573** [astro/deadnix#58]
I. e. this block is transformed via deadnix as follows currently:

```diff
@@ -217,9 +217,6 @@
     let
       rootPath = self;
       forEachSystem = nixpkgs.lib.genAttrs [ "aarch64-linux" "x86_64-linux" ];
-      flakeLib = import ./flake {
-        inherit inputs rootPath forEachSystem;
-      };

       formatterPackArgsFor = forEachSystem (system: {
         inherit nixpkgs system;
@@ -236,7 +233,7 @@
       });

       inherit (nixpkgs.lib) listToAttrs;
-      inherit (flakeLib) mkApp mkHome mkNixOnDroid mkNixos mkDevenvJvmLang mkDevenvDeno mkDevenvFlutter mkDevenvJupyter mkDevenvRuby mkDevenvOcaml;
+      inherit (flakeLib) mkApp mkHome mkNixOnDroid mkNixos mkDevenvJvmLang mkDevenvDeno mkDevenvFlutter mkDevenvRuby mkDevenvOcaml;
     in
     {
       homeConfigurations = listToAttrs [
```

**Comment by astro** [comment]
Please test

**Comment by astro** [comment]
Fixed in 1.2.0
